src/components/AddBook.jsx

```diff
@@ -28,8 +29,8 @@ const AddBook = () => {
 
   return (
     <>
-      <div className="form-container">
-        <h2>Add a new Book</h2>
+      <div className={styles.formContainer}>
+        <h2 className={styles.formTitle}>Add a new Book</h2>
         <form onSubmit={handleSubmit}>
           <input
```

Comment:
- [ ] Dear @Ellyboi, for the add book section, Your design seems to not follow the structure of the rest of the page. In the original design, this section follows the left and right margins of the list of books. Please take a look at the difference and make appropriate changes. Remember as a frontend engineer we are expected to deliver the exact copy of the original design. 

### Your design 
![image](https://github.com/Ellyboi/bookstore/assets/69077061/e5ca0622-7436-4675-9523-bad585e0e8fa)
### Original/expected design 
![image](https://github.com/Ellyboi/bookstore/assets/69077061/f2ef3e4e-3418-4cc5-bc99-4a59d6a69da2)